Library/Formula/ruby.rb

```diff
@@ -56,7 +58,7 @@ def install
       args << "--with-arch=#{Hardware::CPU.universal_archs.join(",")}"
     end
 
-    args << "--program-suffix=23" if build.with? "suffix"
+    args << "--program-suffix=#{program_suffix}" if build.with? "suffix"
```

Comment:
Sorry, I really thought I'd replied here but evidently not. Not a deal-breaker, your point makes total sense.